integration/fuse/src/test/java/alluxio/fuse/cli/MockFuseFileSystemMasterClient.java

```diff
@@ -0,0 +1,214 @@
+      throws AlluxioStatusException {
+    return null;
+  }
+
+  @Override
+  public long getNewBlockIdForFile(AlluxioURI path) throws AlluxioStatusException {
+    return 0;
+  }
+
+  @Override
+  public List<SyncPointInfo> getSyncPathList() throws AlluxioStatusException {
+    return null;
+  }
+
+  @Override
+  public void iterateStatus(AlluxioURI path, ListStatusPOptions options,
+      Consumer<? super URIStatus> action) throws AlluxioStatusException {
+  }
+
+  @Override
+  public List<URIStatus> listStatus(AlluxioURI path, ListStatusPOptions options)
+      throws AlluxioStatusException {
+    return null;
+  }
+
+  @Override
+  public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountPOptions options)
+      throws AlluxioStatusException {
+  }
+
+  @Override
+  public void updateMount(AlluxioURI alluxioPath, MountPOptions options)
+      throws AlluxioStatusException {
+  }
+
+  @Override
+  public Map<String, MountPointInfo> getMountTable() throws AlluxioStatusException {
+    return null;
+  }
+
+  @Override
+  public void rename(AlluxioURI src, AlluxioURI dst) throws AlluxioStatusException {
+  }
+
+  @Override
+  public void rename(AlluxioURI src, AlluxioURI dst, RenamePOptions options)
+      throws AlluxioStatusException {
+  }
+
+  @Override
+  public AlluxioURI reverseResolve(AlluxioURI ufsUri) throws AlluxioStatusException {
+    return null;
+  }
+
+  @Override
+  public void setAcl(AlluxioURI path, SetAclAction action, List<AclEntry> entries,
+      SetAclPOptions options) throws AlluxioStatusException {
+  }
+
+  @Override
+  public void setAttribute(AlluxioURI path, SetAttributePOptions options)
+      throws AlluxioStatusException {
+  }
+
+  @Override
+  public void startSync(AlluxioURI path) throws AlluxioStatusException {
+  }
+
+  @Override
+  public void stopSync(AlluxioURI path) throws AlluxioStatusException {
+  }
+
+  @Override
+  public void scheduleAsyncPersist(AlluxioURI path, ScheduleAsyncPersistencePOptions options)
+      throws AlluxioStatusException {
+  }
+
+  @Override
+  public void unmount(AlluxioURI alluxioPath) throws AlluxioStatusException {
+  }
+
+  @Override
+  public void updateUfsMode(AlluxioURI ufsUri, UpdateUfsModePOptions options)
+      throws AlluxioStatusException {
+  }
+
+  @Override
+  public List<String> getStateLockHolders() throws AlluxioStatusException {
+    return Collections.EMPTY_LIST;
+  }
+
+  @Override
+  public void connect() throws IOException {
+  }
+
+  @Override
+  public void disconnect() {
+  }
+
+  @Override
+  public InetSocketAddress getAddress() throws UnavailableException {
+    return null;
+  }
+
+  @Override
+  public InetSocketAddress getConfAddress() throws UnavailableException {
+    return null;
+  }
+
+  @Override
+  public boolean isConnected() {
+    return false;
+  }
+
+  @Override
+  public boolean isClosed() {
+    return false;
+  }
+
+  @Override
+  public void close() throws IOException {
```

Comment:
Just curious, is there any default one proper? If not necessary, can it be removed?